diff -up qTox-af02542e05992bf94fcff37c365f638ad7b53d8d/CMakeLists.txt.disable_Werror qTox-af02542e05992bf94fcff37c365f638ad7b53d8d/CMakeLists.txt
--- qTox-af02542e05992bf94fcff37c365f638ad7b53d8d/CMakeLists.txt.disable_Werror	2019-09-17 09:49:27.000000000 +0200
+++ qTox-af02542e05992bf94fcff37c365f638ad7b53d8d/CMakeLists.txt	2019-09-22 20:30:44.003786541 +0200
@@ -91,7 +91,6 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}
 set(POSITION_INDEPENDENT_CODE True)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wstrict-overflow")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wstrict-aliasing")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
 
 # avoid timestamps in binary for reproducible builds, not added until GCC 4.9
 include(CheckCXXCompilerFlag)
