diff -up qTox-1.12.1/CMakeLists.txt.disable_Werror qTox-1.12.1/CMakeLists.txt
--- qTox-1.12.1/CMakeLists.txt.disable_Werror	2017-10-11 23:49:18.000000000 +0200
+++ qTox-1.12.1/CMakeLists.txt	2017-10-13 00:40:35.822438119 +0200
@@ -53,7 +53,6 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}
 set(POSITION_INDEPENDENT_CODE True)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wstrict-overflow")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wstrict-aliasing")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
 
 if (NOT WIN32)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fstack-protector-all")
